spring-cloud-dataflow-docs/src/main/asciidoc/api-guide.adoc

```diff
@@ -144,46 +128,112 @@ include::{snippets}/api-documentation/index/links.adoc[]
 [[resources-about]]
 === Server Meta Information
 
+==== Retrieving information about the server
 A `GET` request will return meta information for Spring Cloud Data Flow. This includes:
 
 * Runtime Environment Information
 * Information regarding which features are enabled
 * Dependency information of Spring Cloud Data Flow Server
 * Security information
 
-==== Request structure
+===== Request structure
 
 include::{snippets}/about-documentation/get-meta-information/http-request.adoc[]
 
-==== Request parameters
+===== Request parameters
 
 include::{snippets}/about-documentation/get-meta-information/request-parameters.adoc[]
 
-==== Example request
+===== Example request
 
 include::{snippets}/about-documentation/get-meta-information/curl-request.adoc[]
 
-==== Response structure
+===== Response structure
 
 include::{snippets}/about-documentation/get-meta-information/http-response.adoc[]
 
-[[resources-app-registry-list]]
-=== Listing Applications
+=== Registered Applications
 
+[[resources-app-registry-list]]
+==== Listing Applications
 A `GET` request will list all applications known to Spring Cloud Data Flow.
 
-==== Request structure
+===== Request structure
 
 include::{snippets}/app-registry-documentation/get-applications-filtered/http-request.adoc[]
 
-==== Request parameters
+===== Request parameters
 
 include::{snippets}/app-registry-documentation/get-applications-filtered/request-parameters.adoc[]
 
-==== Example request
+===== Example request
 
 include::{snippets}/app-registry-documentation/get-applications-filtered/curl-request.adoc[]
 
-==== Response structure
+===== Response structure
 
 include::{snippets}/app-registry-documentation/get-applications-filtered/http-response.adoc[]
+
+[[api-guide-resources-stream-definitions]]
+=== Stream Definitions
+
+==== Example "stream create" request for a `ticktock` stream
+
+[source,bash]
+----
+curl -X POST -d "name=ticktock&definition=time | log" localhost:9393/streams/definitions?deploy=false
```

Comment:
Now that I see the port `9393` port is being used here, it might be worth using the same port across the REST documentation instead of `8080`.